scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -324,10 +343,11 @@ Based on our exploration, here is a schematic of how one of the SCE objects will
 
 
 We'll write a _custom function_ (seen in the chunk below) tailored to our wrangling steps that prepares a single SCE object for merging.
-We'll then use our new `purrr::map()` programming skills to run this function over the `sce_list` to end up with a formatted version of `sce_list` that we can merge.
-It's important to remember that this is not a function for general use - it's been precisely written to match the processing we need to do for _these_ SCEs, and different SCEs in the wild will require different types of processing.
+We'll then use our new `purrr::map()` programming skills to run this function over the `sce_list`.
+This will give us a new list of formatted SCEs that we can proceed to merge.
+It's important to remember that the `format_sce()` function written below is not a function for general use - it's been precisely written to match the processing we need to do for _these_ SCEs, and different SCEs in the wild will require different types of processing.
```

Comment:
I did it elsewhere, so I did it here too. This is an en-dash to em-dash change. You can't see it, but it is there. 

```suggestion
It's important to remember that the `format_sce()` function written below is not a function for general use – it's been precisely written to match the processing we need to do for _these_ SCEs, and different SCEs in the wild will require different types of processing.
```